driver.go

```diff
@@ -131,6 +131,144 @@ func (s *Session) processTriple(tx neo4j.Transaction, edge graph.Edge, nodeIds m
 	return nil
 }
 
+func (s *Session) LoadEntityNodes(tx neo4j.Transaction, grph graph.Graph, rootIds []int64, config Config) (graph.Graph, error) {
```

Comment:
Let's change this to  LoadEntityNodes(tx neo4j.Transaction, grph graph.Graph, rootIds []int64, config Config, selectEntity func(graph.Node) bool) (graph.Graph, error)

When you hit an entity node, call selectEntity with it, if it returns true, load that entity as well.